sdk/monitor/Azure.Monitor.Ingestion/api/Azure.Monitor.Ingestion.net461.cs

```diff
@@ -25,6 +26,12 @@ public partial class UploadLogsError
         public Azure.ResponseError Error { get { throw null; } }
         public System.Collections.Generic.IEnumerable<object> FailedLogs { get { throw null; } }
     }
+    public partial class UploadLogsOptions
+    {
+        public UploadLogsOptions() { }
+        public int MaxConcurrency { get { throw null; } set { } }
+        public Azure.Core.Serialization.ObjectSerializer ObjectSerializer { get { throw null; } set { } }
```

Comment:
nit: call the property Serializer

We used that naming in Search - https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs#L81